openrtb_ext/request_wrapper_test.go

```diff
@@ -706,6 +706,7 @@ func TestCloneUserExt(t *testing.T) {
 				eids[1].UIDs[0].AType = 0
 				eids[0].UIDs = append(eids[0].UIDs, openrtb2.UID{ID: "Z", AType: 2})
 				eids = append(eids, openrtb2.EID{Source: "Blank"})
+				// TODO: double-check why we fill eids but don't use it
```

Comment:
@hhhjort

Reply:
(TODO(dmitris) - still open, wait for @hhhjort's response)

Reply:
Can I get an update on this TODO please?

Reply:
I admit I don't fully understand this test - but I added the nolint comment:
`eids = append(eids, openrtb2.EID{Source: "Blank"}) //nolint: ineffassign, staticcheck // this value of `eids` is never used (staticcheck)`
and remove TODO. /cc @hhhjort if you want to double-check it works as intended.

Reply:
Oh, yes, the clone tests. We start with a userext object and a manual copy of that object. (copy/paste of the code lines) We then make a clone of that object using the clone code we wish to test. Next we "mutate" the original, edit it first starting from the outermost reaches, working in towards the root trying to catch a change that the clone code does not protect against. Above we first edit the objects in the UIDs array, in case the clone array has pointers to the items. Then we append, which should replace the array itself, to make sure this doesn't somehow leak over to the clone.

In the end we make all sorts of changes to the original object. We don't care about those shanges, we aren't testing our ability to make these changes. We are only testing that the clone remains independent of all those changes, that the clone is still equal to the manual copy. Thus we get warnings that we don't use the edits we made to the original userext.

Reply:
> Can I get an update on this TODO please?
@AlexBVolcy 

It has been removed, the code now:
```go
			mutator: func(t *testing.T, userExt *UserExt) {
				eids := *userExt.eids
				eids[0].UIDs[1].ID = "G2"
				eids[1].UIDs[0].AType = 0
				eids[0].UIDs = append(eids[0].UIDs, openrtb2.UID{ID: "Z", AType: 2})
				eids = append(eids, openrtb2.EID{Source: "Blank"}) //nolint: ineffassign, staticcheck // this value of `eids` is never used (staticcheck)
				userExt.eids = nil
			},
```